lib/foreman_ansible/engine.rb

```diff
@@ -66,6 +66,7 @@ class Engine < ::Rails::Engine
       ::Api::V2::HostgroupsController.include ForemanAnsible::Api::V2::HostgroupsControllerExtensions
       ::Api::V2::HostgroupsController.include ForemanAnsible::Api::V2::HostgroupsParamGroupExtensions
       ::ConfigReportImporter.include ForemanAnsible::AnsibleReportImporter
+      ::Api::V2::JobTemplatesController.prepend ForemanAnsible::Api::V2::JobTemplatesControllerExtensions
       ::Api::V2::JobTemplatesController.include ForemanAnsible::Api::V2::JobTemplatesControllerExtensions
```

Comment:
The line with `include` should be removed.

Reply:
And it won't be needed if we update the db seed file instead. See the comment ^^

Reply:
> The line with include should be removed.

If I remove it then the `update_api` method in this module won't be included.